src/tactics.ml

```diff
@@ -99,7 +99,8 @@ let handle_tactic : sig_state -> Proof.t -> p_tactic -> Proof.t =
         | 0 -> fst (Scope.scope_term StrMap.empty ss env t)
         | _ -> add_wilds (Pos.none (P_Appl(t, Pos.none P_Wild))) (n-1)
       in
-      handle_refine (add_wilds t nb)
+      let res = (add_wilds t nb) in
```

Comment:
Only change things when necessary. This should be undone.

Reply:
Done.